ci/cirrus.sh

```diff
@@ -25,14 +21,19 @@ sed -i'' -e 's@/usr/bin/file@$(which file)@g' configure
     --with-valgrind="$WITH_VALGRIND" \
     --host="$HOST" $EXTRAFLAGS
 
+make -j2
+
+# Print information about binaries so that we can see that the architecture is correct
+file ./*tests || true
+file ./bench_* || true
+
 if [ -n "$BUILD" ]
 then
-    make -j2 "$BUILD"
+    make "$BUILD"
```

Comment:
Why remove `-j2`?

Reply:
Oh yeah, so the call to `make` doing the actual compilation now is a few linesabove.

But yeah, `$BUILD` is usually `check` here and this can run in parallel too: https://www.gnu.org/software/automake/manual/html_node/Parallel-Test-Harness.html So we should probably do this. In fact, we should have maybe have -j5 or set it cleverly because on Linux we ask for 1 CPU but on MacOS we always get 4 CPUs.

Reply:
I changed the existing commit not to remove -j2 and I added a commit that improves parallelism on macOS. This can be improved further when we overhaul the CI script.